build-infrastructure/audit-logs-stack.yml

```diff
@@ -0,0 +1,121 @@
+---
+AWSTemplateFormatVersion: "2010-09-09"
+Description: A template that creates an audit log and associated logs and buckets
+
+Parameters:
+  KeyAccessTrailName:
+    Type: String
+    Description: The name of the CloudWatch Trail that stores the audit logs
+    Default: key-access-trail
+  KeyAccessAuditLogGroupName:
+    Type: String
+    Description: The name of the log group to push audit logs to
+    Default: key-access-logs
+  LogGroupRetentionPeriodInDays:
+    Type: Number
+    Description: The number of days to retain cloudwatch logs
+    Default: 180
+    AllowedValues:
+      - 1
+      - 3
+      - 5
+      - 7
+      - 14
+      - 30
+      - 60
+      - 90
+      - 120
+      - 150
+      - 180
+      - 365
+      - 400
+      - 545
+      - 731
+      - 1827
+      - 3653
```

Comment:
Why do we need this specific list?

Reply:
https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html
...never mind answered my own question.
maybe a comment about this?